package.json

```diff
@@ -11,7 +11,7 @@
     "gulp": "3.8.7",
     "gulp-filelog": "0.4.1",
     "gulp-include": "1.1.1",
-    "gulp-sass": "1.3.3",
+    "gulp-sass": "2.0.4",
```

Comment:
Last week I had to downgrade my version of node because `gulp-sass` 1.3.3  wasn't recent enough for it.
@tombye mentioned that Travis only runs node v0.10.* or something.  

Do you know if this runs everywhere it's supposed to?